ret-core/src/main/kotlin/io/rabobank/ret/configuration/version/MdcConfig.kt

```diff
@@ -1,10 +1,16 @@
 package io.rabobank.ret.configuration.version
 
 import io.quarkus.runtime.StartupEvent
-import org.jboss.logging.MDC
 import jakarta.enterprise.context.ApplicationScoped
 import jakarta.enterprise.event.Observes
+import org.jboss.logging.MDC
 
+/**
+ * MDC configuration
+ *
+ * Sets the version properties into the MDC, to enrich the log messages.
+ * Typically, you don't use this explicitly when creating a plugin.
```

Comment:
Maybe good to mention a valid use-case?

Reply:
I've rewritten it to actually say you don't need to use it at all